src/config.js

```diff
@@ -113,6 +113,17 @@ export default class Config {
 		return this._getFromSource( this._config, name );
 	}
 
+	/**
+	 * Iterable interface.
+	 *
+	 * Iterates over all top level configuration names.
+	 *
+	 * @returns {Iterable.<String>}
+	 */
+	[ Symbol.iterator ]() {
```

Comment:
If it's iterating over names I'd create it under `Config#names()` or `Config#keys()` depending on how we call this thing. Similar to `Map#keys()` vs `Map#iterator` where the former iterates over prop names and the other over all entries.

Reply:
Done. I added a `Config#names()` method.